core/indexer/elasticsearch.go

```diff
@@ -162,6 +164,8 @@ func (ei *elasticIndexer) createIndex(index string, body io.Reader) error {
 		res, err = ei.db.Indices.Create(index)
 	}
 
+	defer closeESResponseBody(res)
```

Comment:
I would move this after the next check anyway to keep the same format as it used in the other modifications